### ISSUE TITLE
Raise elevation and house separation of Rockmite half wave dipole

### ISSUE THREAD

**Issue by hcarter333** [hcarter333/rockmite#19]
Should all issue titles rhyme?

Raise the elevation and seperation (from the house) of the Rockmite half wave dipole by mounting a twine pulley system to a broken cable clip higher up the wall of the house.

**Comment by hcarter333** [comment]
Furthest contact yet on new antenna:

![image](https://user-images.githubusercontent.com/363004/208894445-2092ac26-402d-4f2c-99fb-39b457f8a4e7.png)

**Comment by hcarter333** [comment]
The antenna re-situation is complete:
The gang and I worked on getting the Rockmite's half-wave dipole a little higher up and a little further away from the house, and in addition to the fun of trying to decide if leaving a fishing rod permanently twined to the house was simply kitschy, or a bit too much [it seems the antenna is just working better](https://www.blogger.com/blog/post/edit/2269351477810212131/5288020208600647323#)!


 The Rockmite made it's furthest away spot at this location! Last night while (trying to?) participating in the [NAQCC sprint](https://www.blogger.com/blog/post/edit/2269351477810212131/5288020208600647323#) the station was heard in Kansas, the far side of of Kansas, Tonganoxie to be exact!
The project started from this sketch:



More details: there's a piece of cable (maybe for an old satellite dish?) on the rear wall of the house. The cable happens to have broken away from one of its plastic support brackets at one point, and that's how all that started. Where other people might see 'broken bracket', the QRPer sees 'antenna mount!' The idea was to plalce a twine loop around the brooken bracket with a piece of much longer twine running through the loop. In this way, the loop becomes a pulley for the longer piece of twine.
To get the loop on the bracket, we put our colllapsible fishing rod into use. I craned out the window and, after several atempts, hooked the loop around the bracket using an open swivel on the end of the fishing rod. Much fun was had by asll as the gang also got to check out the back of the house through the window. Oh, and at one point, we (really I) wound up with the fishing rod stuck in the pulley loop. The consideration of just leaving the thing there really did come up.

Next, we put the end of the pulley string around our antenna stress relief, (read BNC to bannana connector). At that point, the 11 year old went down to the yard, disconnected the two twine ends of the existing antenna nad tied them further out along the fence line, (patiently putting up with my not always clear instructions as I inutited what we were going to do.)

We had an almost immedate payoff that night as the station was spotted in Canada with 5 dB signal to noise ratio at the latest time we've achieved there. The following day, the 20th, we achieved the highest SNR we've seen at 17 dB into Arizona.

We need more data to make suer the bands didn't just get better, but the new antenna is looking good!
